libkbfs/test_util.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
```

Comment:
It would, except that has several exported pieces that are used in tests by other libraries (such as the DSL tests).  Probably there are some unexported things that could be moved out of there.